man/journalctl.xml

```diff
@@ -92,8 +92,8 @@
     disabled; see the <option>--no-pager</option> option and the "Environment" section below.</para>
 
     <para>When outputting to a tty, lines are colored according to priority: lines of level ERROR and higher
-    are colored red; lines of level NOTICE and higher are highlighted; lines of level DEBUG are colored
-    lighter grey; other lines are displayed normally.</para>
+    are colored red; lines of level WARNING are colored yellow; lines of level NOTICE and higher are highlighted;
+    lines of level DEBUG are colored lighter grey; other lines are displayed normally.</para>
```

Comment:
hmm, if you touch this, can you drop the "lighter" grey? it's actually a middle grey, since it out to work on both bright and dark backgrounds. hence just drop the word, and just say "grey".